components/hab/src/cli.rs

```diff
@@ -905,6 +906,36 @@ fn sub_pkg_build() -> App<'static, 'static> {
     sub
 }
 
+fn sub_pkg_download(feature_flags: FeatureFlag) -> App<'static, 'static> {
+    let mut sub = clap_app!(@subcommand download =>
+    (about: "Download Habitat artifacts (including dependencies and keys) from Builder")
+    (@arg AUTH_TOKEN: -z --auth +takes_value "Authentication token for Builder")
+    (@arg BLDR_URL: --url -u +takes_value {valid_url} default_value(habitat_core::url::DEFAULT_BLDR_URL)
+        "Specify an alternate Builder endpoint. If not specified, the value will \
+         be taken from the HAB_BLDR_URL environment variable if defined. (default: \
+         https://bldr.habitat.sh)")
+    (@arg CACHE_DIRECTORY: --cache +takes_value "The path to store downloaded artifacts")
+    (@arg CHANNEL: --channel -c +takes_value default_value[stable] env(ChannelIdent::ENVVAR)
+        "Install from the specified release channel (default: stable)")
+    (@arg PKG_IDENT_FILE: --file +takes_value +multiple
+        "File with newline separated package identifiers")
+    (@arg PKG_IDENT: +multiple
+            "One or more Habitat package identifiers (ex: acme/redis)")
+    (@arg PKG_TARGET: --target -t +takes_value {valid_target}
+            "Target architecture to fetch. Allowable options are one of (TODO ADD THIS)")
+    );
+    // TODO: Discover the background for this; while it seems reasonable to have, I'd really like to
+    // not cargo cult it in.
+    if feature_flags.contains(FeatureFlag::IGNORE_LOCAL) {
```

Comment:
Don't think we need this..

Reply:
For sure; in no way do we want to fall back to trying to resolve idents from the cache. In other words, IGNORE_LOCAL=true is the only correct semantic I thin.